src/tools/ip/quicip.h

```diff
@@ -135,7 +143,7 @@ MsQuicGetPublicIPEx(
     const QUIC_BUFFER Alpn = { sizeof("ip") - 1, (uint8_t*)"ip" };
     const uint16_t UdpPort = 4444;
 
-    QUIC_SETTINGS Settings{0};
+    QUIC_SETTINGS Settings = {0};
```

Comment:
I don't think these are equivalent in C++. I believe the old way will correctly zero initialize all members of the struct, while the new will (on some compilers) only zero-initialize the first member.